control-plane-operator/hostedclusterconfigoperator/controllers/resources/ingress/params.go

```diff
@@ -9,16 +10,30 @@ type IngressParams struct {
 	IngressSubdomain string
 	Replicas         int32
 	PlatformType     hyperv1.PlatformType
+	IsPrivate        bool
+	CloudUPI         bool
 }
 
 func NewIngressParams(hcp *hyperv1.HostedControlPlane) *IngressParams {
 	var replicas int32 = 1
+	isPrivate := false
+	CloudUPI := false
+	if hcp.Spec.Platform.IBMCloud != nil && hcp.Spec.Platform.IBMCloud.ProviderType == configv1.IBMCloudProviderTypeUPI {
+		CloudUPI = true
```

Comment:
nit: why is this variable exported? CloudUPI -> cloudUPI or make it ibm specific ibmCloudUPI